zebra-consensus/src/router/service_trait.rs

```diff
@@ -0,0 +1,15 @@
+//! Trait aliases for block verification Tower services.
+//!
+//! This trait provides a convenient alias for `tower::Service`
+//! implementations that operate on Zebra block verification request and response types.
+//!
+//! - [`BlockVerifierService`]: for services that handle semantic block verification requests.
+
+use crate::router::Request;
+use zebra_chain::block::Hash;
+use zebra_node_services::service_traits::ZebraService;
+
+/// Trait alias for services handling semantic block verification requests.
```

Comment:
Nitpick: The service may also route verification requests to the checkpoint verifier and won't always semantically verify the blocks.

```suggestion
//! - [`BlockVerifierService`]: for services that handle block verification requests.

use crate::router::Request;
use zebra_chain::block::Hash;
use zebra_node_services::service_traits::ZebraService;

/// Trait alias for services handling block verification requests.
```

Reply:
Done at https://github.com/ZcashFoundation/zebra/pull/10053/commits/ad647b16325ebf38711f577cfbf1ca00500e43dd